### ISSUE TITLE
windows install file has Win32:Malware-gen Viruses 

### ISSUE THREAD

**Issue by yaohaixiao** [flagbug/Espera#146]
Avast reports the install file has Viruses

**Comment by flagbug** [comment]
Yeah, that's a false positive.

See [here](https://github.com/Squirrel/Squirrel.Windows/issues/218) and [here])(https://github.com/atom/atom/issues/3927), other people have the same problem with the installer I'm using

**Comment by paulcbetts** [comment]
@flagbug Are you signing the installer? If you can do it, signing the installer with a code signing cert will vastly increase the confidence AV vendors have that it's not a virus

**Comment by flagbug** [comment]
@paulcbetts I'm a poor OSS developer I can't afford one of those :smile:

**Comment by paulcbetts** [comment]
@flagbug I think that prices for them have dropped significantly, it used to be crazy $$ but now you can get them on the cheap

**Comment by flagbug** [comment]
@paulcbetts I think that's only true for SSL certificates, Code Signing certificates are still super expensive (well, at least it's too expensive for a hobby project that generates no income)

**Comment by flagbug** [comment]
Seems like you can get a free certificate for open source projects from some authorities.
Just got mine, so this should be fixed soon!

**Comment by paulcbetts** [comment]
virustotal.com is a good indicator to tell you how likely you are to run into AV trouble

**Comment by flagbug** [comment]
@paulcbetts Yup, I already know about virustotal, but thanks for the info anyway!

On another note, I signed the installer, but Avast still whines that it's malware :crying_cat_face:

**Comment by Panthro** [comment]
Hey @flagbug I'm having the same issue with an electron app I'm building, how did you get the certificate?

Also, have you finally fixed it?
